### PR TITLE
Update x11.cc

### PR DESCRIPTION
**Pull request by chepioq**
Solve issue #284 "Alignment abbreviations not working"

### CODE REVIEW

**Comment by chepioq** [comment]
Nobody for push this commit to the master branch ?

**Comment by MattSturgeon** [comment]
> Nobody for push this commit to the master branch ?

Don't be offended, @brndnmtthws doesn't seem to have time to look at conky very often, I'm sure it's nothing personal!

As a side note, it may be worth writing _slightly_ more descriptive commit messages/PR descriptions. A better description might be something like "The conky docs for `alignment` say that abreviations can be used, however they are not currently recognised by `X11.cc` (see #284). This fixes #284 by adding the listed abbreviations to the `alignment` arguments map."

**Comment by chepioq** [comment]
Thank MattSturgeon for your comment.
I am not offended and I agree with you.
I copy your description to #284 and after wait and see.
Thank you again.

File: src/x11.cc [hunk 1]
```diff
@@ -246,6 +246,15 @@ conky::lua_traits<alignment>::Map conky::lua_traits<alignment>::map = {
...
 	{ "middle_right",  MIDDLE_RIGHT },
+	{ "tl",            TOP_LEFT },
+	{ "tr",            TOP_RIGHT },
+	{ "tm",            TOP_MIDDLE },
+	{ "bl",            BOTTOM_LEFT },
+	{ "br",            BOTTOM_RIGHT },
+	{ "bm",            BOTTOM_MIDDLE },
+	{ "ml",            MIDDLE_LEFT },
+	{ "mm",            MIDDLE_MIDDLE },
+	{ "mr",           MIDDLE_RIGHT },
```
Comment (brndnmtthws) [inline src/x11.cc]: Alignment is off here.

**Comment by brndnmtthws** [comment]
One small comment. Address that, and we should be good to go.

**Comment by chepioq** [comment]
What do you mean by "Address that" ?
I can see you don't update Update x11.cc, and alignment abbreviations not working in this branch.

**Comment by Nooby4Ever** [comment]
That means if you update the alignment in your code, as stated by brndnmtthws (MIDDLE_RIGHT is out of sync),  eg in a new commit he will merge this Pull Request to the master.

**Comment by chepioq** [comment]
Ok I close this pull request and open a new one with good information.

**Pull request closed without merging**